apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -97,6 +114,79 @@ class ConnectionInstructions extends React.Component {
   }
 }
 
+class CPExpressInstallInstructions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // TODO: layout with images
+    // TODO: get screenshot for image copy
+    // TODO: reset button gif- put in our cdn?
+    // TODO: make i18n strings for all the strings
+    return (
+      <div id="CPExpressInstallInstructions">
+        <h2>Install the Firmata firmware onto the CP Express</h2>
+        <ol>
```

Comment:
First step should be to connect the CP Express to your computer using a USB cable just in case a user jumps to this section before going to the Checklist above.